packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -884,8 +888,8 @@ Future<T> showCupertinoModalPopup<T>({
   );
 }
 
-final Animatable<double> _dialogTween = Tween<double>(begin: 1.2, end: 1.0)
-  .chain(CurveTween(curve: Curves.fastOutSlowIn));
+final Animatable<double> _dialogTween = Tween<double>(begin: 1.3, end: 1.0)
```

Comment:
ditto

Reply:
While you're here, can you rename this _dialogScaleTween? Since it's not clear what this does by the name. Was the 1.2->1.3 change intentional?

Reply:
Will do. I bumped the initial scale to 1.3 to compensate for the higher initial velocity of the new curve. I can add another comment here if that would be best.

Reply:
Sure. Don't comment on the difference but do add a comment on how the final value was derived.

Reply:
Done